### DISCUSSION TITLE
Dedicated GPU support (Win 10 21H1, Nvidia Quadro M2200)

### DISCUSSION THREAD

**Discussion by rounakarya** [rocksdanister/lively#577] [Q&A]
Tried everything to run the app on the Nvidia Quadro M2200. Changed the Windows 10 Graphics settings and Nvidia control panel for both the EXEs (both livelywpf.exe & plugins/cef/LivelyCefSharp.exe) 
The app is still seen running on the Integrated Intel GPU.
Any solutions?

**Comment by rocksdanister** [comment]
hmm maybe its some power limitation? try plug into socket.

**Reply by rounakarya** [reply]
My laptop was plugged in the whole time.

**Reply by rocksdanister** [reply]
Is it the same with the latest Webview plugin?
https://github.com/rocksdanister/lively/releases/tag/v1.7.0.0

**Reply by rounakarya** [reply]
Yes, I tried with the latest version(1.7.0.0) and I am using the default wallpapers like Fluid v2.

**Reply by rocksdanister** [reply]
Did you turn the Edge webview on in settings?

**Reply by rounakarya** [reply]
I just installed Edge webview and switched the Web Browser Engine to MS Edge and that threw an error. 
Wallpaper set was Fluid v2.

Here is the log file:
[20210627_132535.txt](https://github.com/rocksdanister/lively/files/6721232/20210627_132535.txt)

**Reply by rocksdanister** [reply]
Yeah looks like if you do admin install (program files (x86\..) then webview2 will fail as its trying to create cache folder there; need to change it.

Made an issue tracker https://github.com/rocksdanister/lively/issues/584
